static/js/components/RepeatableContentListing.tsx

```diff
@@ -37,16 +38,32 @@ import SiteContentEditorDrawer from "./SiteContentEditorDrawer"
 import { useWebsite } from "../context/Website"
 import { formatUpdatedOn } from "../util/websites"
 import Dialog from "./Dialog"
+import posthog from "posthog-js"
 
 export default function RepeatableContentListing(props: {
   configItem: RepeatableConfigItem
 }): JSX.Element | null {
   const store = useStore()
   const { configItem } = props
   const isResource = configItem.name === "resource"
-  const isExternalResource = configItem.name === "external-resource"
+
   const website = useWebsite()
 
+  const [isContentDeletable, setIsContentDeletable] = useState(false)
+
+  useEffect(() => {
+    const checkFeatureFlag = async () => {
+      const flagEnabled =
+        posthog.isFeatureEnabled("OCW_STUDIO_CONTENT_DELETABLE") ?? false
```

Comment:
This checks for feature flag without any user id verification. It'd be a good idea to send user information as well, so that we may control the rollout of feature to specific users. keeping that in mind, I'd suggest the following change.

```suggestion 
        posthog.isFeatureEnabled("OCW_STUDIO_CONTENT_DELETABLE", SETTINGS.user.email ) ?? false
```

Reply:
Let's do this instead https://github.com/mitodl/ocw-studio/pull/2291#discussion_r1747231345